packages/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -47,7 +47,7 @@ export const useFluentProvider = (
    * see https://github.com/microsoft/fluentui/blob/0dc74a19f3aa5a058224c20505016fbdb84db172/packages/fluentui/react-northstar/src/utils/mergeProviderContexts.ts#L89-L93
    */
   state.targetDocument = state.targetDocument ?? parentContext.targetDocument;
-  state.dir = state.dir ?? parentContext.dir;
+  state.dir = state.dir ?? parentContext.dir ?? document.dir;
```

Comment:
should this follow the same pattern as `getRTL`? 

`(doc.body && doc.body.getAttribute('dir')) || doc.documentElement.getAttribute('dir')`